docs/developer_guide/advanced_topics/overriding_agg_fn.rst

```diff
@@ -10,7 +10,7 @@ Override Aggregation Function
 With the aggregator-based workflow, you can use custom aggregation functions for each task via Python\*\  API or command line interface.
 
 
-Python API
+Python API (Deprecated)
```

Comment:
If we're deleting the entire `openfl.native` package, this section should be deleted too (instead of being deprecated).